### PR TITLE
add line numbers to large codeblocks

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Adding line numbers per code block, rather than globally. It is possible but the generated output looks bad:

Screenshot 2019-08-09 at 12 42 03

Needs further investigation.

# Reasons
A customer has reported that it's hard to follow code walk-throughs when the blocks are long, without line numbers.

### CODE REVIEW

**Comment by rosieyohannan** [comment]
I've realised the way this looks is due to the code block being formatted as a table when line numbers are enabled, so I made them disappear by changing them to the same colour as the background in CSS. Just need to see if I can reduce the space at the top and bottom of the block now, but should have something usable soon...

Screenshot 2019-08-12 at 20 30 41

**Comment by teesloane** [comment]
This looks great!

**Comment by rosieyohannan** [comment]
Done!

Screenshot 2019-08-12 at 21 31 30

**Comment by rosieyohannan** [comment]
I Just need to add this to the rest of the guided code blocks and then ready to merge.

**Comment by rosieyohannan** [comment]
fixes #3563
